store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -239,7 +239,7 @@ public class MessageStoreConfig {
     private boolean recheckReputOffsetFromCq = false;
 
     // Maximum length of topic
-    private int maxTopicLength = 1000;
+    private int maxTopicLength = 127;
```

Comment:
It would be better to use `Byte.MAX_VALUE` to represent 127

Reply:
> It would be better to use `Byte.MAX_VALUE` to represent 127

Good idea